test.py

```diff
@@ -9,4 +9,4 @@
 result = engine.execute(sql)
 
 for row in result:
-    print(row[0])
\ No newline at end of file
+    print(row[0])
```

Comment:
I didn't introduce this file. It is used as a simple test program outside of the unit tests just to run and print queries.
Arguably it could be removed.